IOSSecuritySuite/FishHookChecker.swift

```diff
@@ -123,52 +124,53 @@ internal class FishHookChecker {
                     curCmd.pointee.segname.6 == linkeditCmdName[6] &&
                     curCmd.pointee.segname.7 == linkeditCmdName[7] &&
                     curCmd.pointee.segname.8 == linkeditCmdName[8] &&
-                    curCmd.pointee.segname.9 == linkeditCmdName[9]) {
-                    
+                    curCmd.pointee.segname.9 == linkeditCmdName[9] {
+
                     linkeditCmd = curCmd
                 }
-                if (curCmd.pointee.segname.0 == textCmdName[0] &&
+                if curCmd.pointee.segname.0 == textCmdName[0] &&
                     curCmd.pointee.segname.1 == textCmdName[1] &&
                     curCmd.pointee.segname.2 == textCmdName[2] &&
                     curCmd.pointee.segname.3 == textCmdName[3] &&
                     curCmd.pointee.segname.4 == textCmdName[4] &&
-                    curCmd.pointee.segname.5 == textCmdName[5]) {
-                    
+                    curCmd.pointee.segname.5 == textCmdName[5] {
+
                     textCmd = curCmd
                 }
             } else if curCmd.pointee.cmd == LC_DYLD_INFO_ONLY || curCmd.pointee.cmd == LC_DYLD_INFO {
                 dyldInfoCmd = UnsafeMutablePointer<dyld_info_command>(OpaquePointer(UnsafeRawPointer(curCmd)))
             }
         }
-        
+
         if linkeditCmd == nil || dyldInfoCmd == nil || textCmd == nil { return }
-        
+
         let linkeditBase = UInt64(slide) + linkeditCmd.pointee.vmaddr - linkeditCmd.pointee.fileoff
```

Comment:
the `slide` is type of Int, so maybe it's negative value(even not happen for me).
`let linkeditBase = UInt64(slide + Int(linkeditCmd.pointee.vmaddr) - Int(linkeditCmd.pointee.fileoff))` 
may be better